### PR TITLE
Multithreaded HTTP handler to server concurrently

### PR DESCRIPTION
**Pull request by luisferreira32**
A first implementation of the HTTP server was serving clients sequentially.
Now with the multi-threaded HTTP handler it is possible to serve them 
concurrently.

side note: I believe matplotlib might break serving two clients at the same time.
will try to do a proper integration test to simulate this behaviour after #34 is merged.

### CODE REVIEW

**Comment by luisferreira32** [comment]
Found an issue with multi threaded usage of matplotlib. Will take some time to investigate.

**Comment by luisferreira32** [comment]
So.... we can only serve one user at a time? https://matplotlib.org/3.1.0/faq/howto_faq.html#working-with-threads

**Comment by luisferreira32** [comment]
I believe I've figured it out. Left links in the code as well, so that it makes sense where that back-end setting comes from.

File: services/plotter/plotter.py [hunk 1]
```diff
@@ -1,42 +1,44 @@
...
+from matplotlib.figure import Figure
+from matplotlib.axes import Axes
+from pandas import DataFrame
 
-from utils.log import logger
 
+def plotter(data: DataFrame, configs: dict) -> BytesIO:
+    uid = uuid4()
+    fig: Figure = figure(uid)
+    axes: Axes = fig.add_axes([.1, .1, .8, .8])
```
Comment (MrToino) [inline services/plotter/plotter.py:13]: That's great, however, note that the axes object may be more complex than this for multi-axes figures. Later on we should check if this is compatible with multiple axes plot (multiple axes on one same figure). For now it seems ok.

**Review by MrToino — APPROVED** [review]
That's great news! We're on the right track.

File: services/plotter/plotter.py (hunk 1 above)
Comment (luisferreira32) [reply services/plotter/plotter.py:13]: yea, agreed, we have to change this configuration for multi axes in the future

**Pull request merged**